src/lib/features/Header.svelte

```diff
@@ -1,11 +1,11 @@
 <script lang="ts">
-  import { FrequencyAccessLogo, FrequencyLogo, OpenClose } from '../assets/index';
+  import { FrequencyLogo, OpenClose } from '../assets/index';
   import NavMenu from './NavMenu.svelte';
   import NavMenuMobile from './NavMenuMobile.svelte';
   import { MenuItem } from '$lib/utils/types.js';
 
   export let menuItems: MenuItem[] = [];
-  export let logoType: 'primary' | 'access' = 'primary';
+  export let logoType: 'primary' = 'primary';
```

Comment:
if the logo type is always primary, then we no longer need to define that value. Let's remove logoType.

Reply:
Done in the follow up for the header changes